osu.Game.Modes.Taiko/UI/RingExplosion.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Transforms;
+using osu.Game.Graphics;
+using osu.Game.Modes.Taiko.Judgements;
+using osu.Game.Modes.Taiko.Objects;
+
+namespace osu.Game.Modes.Taiko.UI
+{
+    /// <summary>
+    /// A ring that explodes to indicate a judgement has occurred.
+    /// </summary>
+    internal class RingExplosion : CircularContainer
+    {
+        /// <summary>
+        /// The Judgement to display.
+        /// </summary>
+        public TaikoJudgementInfo Judgement;

```

Comment:
probably best to make readonly and provide via `ctor`. right now this can be changed but will have no effect on the drawable